articles/aks/node-updates-kured.md

```diff
@@ -19,7 +19,7 @@ The process to keep Windows Server nodes up to date is a little different. Windo
 This article shows you how to use the open-source [kured (KUbernetes REboot Daemon)][kured] to watch for Linux nodes that require a reboot, then automatically handle the rescheduling of running pods and node reboot process.
 
 > [!NOTE]
-> `Kured` is an open-source project by Weaveworks. Please direct issues to the [kured GitHub][kured]. Additional support can be found in the #weave-community Slack channel.
+> `Kured` is an open-source project in the CNCF. Please direct issues to the [kured GitHub][kured]. Additional support can be found in the #kured channel on CNCF Slack.
```

Comment:
We should spell out this acronym - Cloud Native Computing Foundation (CNCF). Also, replace - found in the #kured channel to - found on the #kured channel on CNCF Slack.

Reply:
Thanks for making the suggested change. However, you forgot to include the acronym with the spelled out term. Apologies if I wasn't specific enough in my initial feedback.